src/openassetio-core/include/openassetio/hostApi/Manager.hpp

```diff
@@ -858,8 +858,8 @@ class OPENASSETIO_CORE_EXPORT Manager {
    * to @ref managementPolicy in order to determine if the manager
    * handles relationships of that type.
    */
-  void getWithRelationship(const TraitsDataPtr& relationshipTraitsData,
-                           const EntityReferences& entityReferences,
+  void getWithRelationship(const EntityReferences& entityReferences,
```

Comment:
[fix] Needs a breaking change in `RELEASE_NOTES.md` for these changes.

Reply:
Sorry, scratch that - I forgot we haven't released since we refactored the method in the first place.